GameData/RealismOverhaul/Waterfall_Configs/MandatoryRCSPartPack/mandatory-rcs.cfg

```diff
@@ -0,0 +1,75 @@
+// N.B.: These blocks are nasty because they manipulate the `MODEL`'s scale directly instead of
```

Comment:
Where is this done?  Don't see it in the patches, which only appear to manipulate the plume scale?

Reply:
It's done here: https://github.com/KSP-RO/RealismOverhaul/tree/master/GameData/RealismOverhaul/RO_SuggestedMods/MandatoryRCSPartPack

Reply:
I see.  If we ever get around to touching those files, may fix.  (I tend to convert things to use rescaleFactor when I'm making other modifications).  Is that relevant to these patches?

Reply:
No, not relevant. I complained here because I didn't expect these blocks to manipulate `MODEL` and figured that I should explain why this patch uses a different strategy than the other blocks.